server.js

```diff
@@ -33,14 +35,17 @@ server.all('/', (req, res) => {
   res.status(302).redirect('https://developers.rsk.co/');
 });
 
+// health check
 server.get('/api/status', (req, res) => {
   res.send({
     ok: Date.now(),
   });
 });
 
+// api router
 server.use('/api/v1', apiRouter);
 
-server.use(express.static(__dirname + '/static'));
```

Comment:
Consider setting `maxage` - I suggest for 1 day, to better use the built-in caching from `express.static`. `etag` is the other important one, but that is enabled by default already.

Reply:
FYI this is a good reference article: https://evanhahn.com/express-dot-static-deep-dive/